app/src/main/res/xml/preferences.xml

```diff
@@ -280,6 +280,10 @@
                 android:defaultValue="false"
                 android:key="display-keyboard"
                 android:title="@string/keyboard_name" />
+            <fr.neamar.kiss.preference.SwitchPreference
+                android:defaultValue="true"
+                android:key="allow-hide-keyboard"
+                android:title="@string/allow_hide_keyboard_title" />
```

Comment:
Would it make sense to disable this option if `Display keyboard on start` is not enabled?
What do you think? 

To me personally it feels somehow weird if keyboard is not shown at start, but can't be hidden anymore if it is shown once later on.